src/commands/troll/hitlist.ts

```diff
@@ -0,0 +1,42 @@
+import { KillIntervals } from "../../custom/storage";
+import {
+    CommandoMessage,
+    Command,
+    CommandoClient,
+    ArgumentInfo,
+} from "discord.js-commando";
+import { MembershipStates, Message, SystemChannelFlags } from "discord.js";
+
+module.exports = class HitListCommand extends (Command) {
+    constructor(client: CommandoClient) {
+        super(client, {
+            name: "hitlist",
+            group: "troll",
+            memberName: "hitlist",
+            description: "Displays killed people in the server in a hitlist.",
+        });
+    }
+
+    async run(message: CommandoMessage, args: any) {
+        return message.say("People currently killed in the server: " + killedPeople());
+    }
+};
+
+
+/**
+ * Returns values of killed users of a server in an array.
+ * @returns {members} array of killed users in a server.
```

Comment:
`{member}` should be the variable type you are returning, not the name of the variable. In this case, you are returning an array of strings. Change this so it knows what you are returning.